routes/curations.js

```diff
@@ -52,10 +52,18 @@ router.get(
 router.get(
   '/:type?/:provider?/:namespace?/:name?',
   asyncMiddleware(async (request, response) => {
-    const coordinates = utils.toEntityCoordinatesFromRequest(request)
-    return curationService.list(coordinates).then(result => response.status(200).send(result))
-  })
-)
+      const coordinates = utils.toEntityCoordinatesFromRequest(request)
+      curationService.list(coordinates).then(function(result){
+         /* HANDLE PROMISE #1 */
+         var text_file_name = "./test/fixtures/curations_dummy_data.json";
```

Comment:
@moranthomas are we sure this is a var? is it not better to use an es6 const in this case.
I would recommend that we remain consistent with the overall codebase, so e.g. use let vs var, const where needed etc.

Reply:
Thanks Alex. I agree. I have updated the code with your suggested edits and re-committed